src/components/sidenav/drawer.scss

```diff
@@ -0,0 +1,27 @@
+@import "variables";
+@import "shadows";
+
+
+:host {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  z-index: $z-index-drawer;
+  box-sizing: border-box;
+
+  background-color: white;
```

Comment:
Actually, it seems the default background is `rgba(0,0,0,0)` which I find awkward.